sdk/appconfiguration/app-configuration/test/index.spec.ts

```diff
@@ -251,6 +260,21 @@ describe("AppConfigurationClient", () => {
 
       assert.equal(result.length, 2);
     });
+
+    it("sample code works", async () => {
+      const key = `listRevisionsSample-${Date.now()}`;
+
+      // create a new setting
+      await client.addConfigurationSetting(key, { value: "foo1" });
+
+      // update it with a new value
+      await client.setConfigurationSetting(key, { value: "foo2" });
+
+      // retrieves all revisions matching that label
+      const result = await client.listRevisions({ key: [key] });
+
+      assert.equal(result.length, 2);
+    })
```

Comment:
Super nit: missing semi-colon. Could you run `npm run format` inside this package after getting approvals? That will run our formatter that should fix things like this (I think I forgot to run it, so you'll likely see a lot of whitespace changes)

Reply:
Done. And yes, there were...a few whitespace changes. :)